made to match husky as a template

jeremy-a opened the pull request:
matching husky as template/starting point

jeremy-a committed:
made to match husky as a template

matching husky as template/starting point

IanTheEngineer commented on microtrac_gazebo/env-hooks/50.microtrac_gazebo.sh:
```diff
@@ -0,0 +1 @@
+export microtrac_GAZEBO_DESCRIPTION=$(catkin_find --without-underlays --first-only microtrac_gazebo urdf/description.gazebo.xacro 2>/dev/null)
```
What is this script trying to accomplish?

jeremy-a replied:
Again from the Husky but I removed it in the latest commit.

IanTheEngineer commented on microtrac_gazebo/CMakeLists.txt:
```diff
@@ -1,6 +1,15 @@
+
 catkin_package()
+catkin_add_env_hooks(50.microtrac_gazebo
+  SHELLS sh DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/env-hooks)
```
Shell scripts in cmake files are a bit funky. Is this to construct the URDF?

jeremy-a replied:
This came from Husky. I removed it in my latest commit in favor of a find in the launch file.

IanTheEngineer commented on microtrac_gazebo/launch/microtrac_empty_world.launch:
```diff
@@ -0,0 +1,49 @@
+  <arg name="world_name" default="worlds/empty.world"/>
+
+  <arg name="laser_enabled" default="true"/>
+  <arg name="ur5_enabled" default="false"/>
```
laser and ur5? These don't seem to apply

jeremy-a replied:
Removed laser, UR5, and kinect throughout in my latest commit.

IanTheEngineer commented on microtrac_gazebo/urdf/microtrac.gazebo.xacro:
```diff
@@ -0,0 +1,95 @@
+    </gazebo>
+
+    <gazebo>
+      <plugin name="gps_controller" filename="libhector_gazebo_ros_gps.so">
```
+1 for the gps ros plugin and IMU.

IanTheEngineer commented on microtrac_gazebo/urdf/microtrac.gazebo.xacro:
```diff
@@ -0,0 +1,95 @@
+      </plugin>
+    </gazebo>
+  
+    <xacro:if value="$(arg ur5_enabled)">
```
Again, excess UR5 stuff here

IanTheEngineer reviewed (changes requested):
@jeremy-a, thank you for showing this repository some love. I do like to see the additions of the GPS and IMU plugins, but in general, the Husky files add a fair amount of excess bloat code. Would you mind pairing things down to the base necessities?

jeremy-a committed:
cleanup plugins which are NA

Laser, UR5, and kinect removed.  Remove use of shell script in favor of package find. Build but  not running.
